organisationsnummer.go

```diff
@@ -183,7 +188,14 @@ func (o *Organisationsnummer) String() string {
 }
 
 // Get vat number for a organization number.
+//
+// Deprecated: vatNumber function was named wrongly, use "VatNumber" instead.
 func (o *Organisationsnummer) vatNumber() string {
```

Comment:
Do we need to keep this? wouldn't the `v` make this function private? (been a short while since I messed with go, so might be wrong ;D )

Reply:
Ah, yeah you're right 👍